composer.json

```diff
@@ -1,15 +1,28 @@
 {
     "name": "mahocommerce/maho-composer-plugin",
+    "description": "Extension for Composer to copy assets and enable autoloading for Maho projects.",
     "type": "composer-plugin",
+    "license": "MIT",
     "require": {
-        "composer-plugin-api": "^2.0"
+        "composer-plugin-api": "^2.1",
+        "composer-runtime-api": "^2"
+    },
+    "require-dev": {
+        "phpstan/phpstan": "^2.0",
+        "phpstan/phpstan-strict-rules": "^2.0",
+        "phpstan/phpstan-deprecation-rules": "^2.0",
```

Comment:
these 2 will get propagated to a maho child project too right? are they necessary?

Reply:
Composer won’t install dev dependencies of a required package, it’s only for this repo.